src/content/guides/scaffolding.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Scaffolding
+contributors:
+  - evenstensberg
+  - pranshuchittora
+sort: 2
+---
+
+It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance is quite hard. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
+
+## Creating a scaffold
+
+Before writing a `webpack-cli` scaffold, think about what you're trying to achieve. Do you want a "general" scaffold that could be used by any project or type of app? Do you want something focused, like a scaffold that writes both your `webpack.config.js` and your framework code? It's also useful to think about the user experience for your scaffold.
+
+`webpack-cli` offers an interactive experience to customize the output accordingly. For example asking questions like: "What is your entry point?".
+
+### Writing a scaffold
+
+There are many resources where you can learn how to write a scaffold, you can start from [Writing a Scaffold](/contribute/writing-a-scaffold/)
+
+`webpack-scaffold` is a utility suite for creating scaffolds. It contains functions that could be used to create a scaffold.
+
+### Running a scaffold
+
+A scaffold can be executed using `webpack-cli init`:
+
+```bash
+webpack-cli init <your-scaffold>
+```
+
+#### Running a scaffold locally
+
+When the scaffold package is in your local file system you should point `init` to its path:
+
+```bash
+webpack-cli init path/to/your/scaffold
+```
+
+Or you can create a global module and symlink to the local one:
+
+- Using npm
+  `bash cd path/to/my-scaffold npm link webpack-cli init my-scaffold`
+- Using yarn
+  `bash cd path/to/my-scaffold yarn link webpack-cli init my-scaffold`
+
+#### Running a scaffold from npm
+
+If the package is available from npm, its name must begin with `webpack-scaffold` and can be used by running:
+
+```bash
+webpack-cli init webpack-scaffold-yourpackage
+```
+
+## API
+
+To create a `scaffold`, you must create a [`yeoman-generator`](http://yeoman.io/authoring/). Thanks to it, you can optionally extend your generator to include methods from the [Yeoman API](http://yeoman.io/learning/). It's worth noting that we support all the properties of a regular webpack configuration. In order for us to do this, there's a thing you need to remember:
+
+W> Objects are made using strings, while strings are made using double strings. This means that in order for you to create a string, you have to wrap it inside another string for us to validate it correctly.
+
+### Required
+
+- [API](#api)
+
```

Comment:
Alignment is lost, please re-align

Reply:
markdown linter is throwing error, that's why it's like this.

Reply:
Okay, i can resolve that later